### DISCUSSION TITLE
How to filter built in map scalar query results by value coresponding to key

### DISCUSSION THREAD

**Discussion by KonradMatusiak** [99designs/gqlgen#1811] [Q&A]
### What happened?
I have an object with dynamic tags. I want to be able to filter the results by a tag key value.
A map fits my needs perfectly and the documentation mentions map scalars. However it doesn't mention how to query it. 
However it doesn't specify how query and filter the results. I expected the following results with the query and schema provided below.
What am I doing wrong? How do I use filters in this case? Thank you in advance.
### expected response
```
query{
  books{
    title
    tags(Genre:"whodunit")
  }
}

{
  "data": {    
    "books": [
      {
        "title": "somecrimestory
        "tags": {
         "Genre": "whodunit"
         "Lenght": "short"
        }
      },
      {
        "title": "somecrimestory
        "tags": {
         "Genre": "whodunit"
         "Lenght": "long"
        }
      }
    ]
  }
}
```
### Schema

```
scalar Map
scalar Time
type Book {
  title: string!
  tags: Map!
}

type Query {
  books: [Book!]!
}
```
### versions
 - `gqlgen version` 0.13.0
 - `go version` 1.13.8
 -  `module`

**Comment by razorness** [comment]
Read this:
https://graphql.org/learn/queries/
